packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -528,6 +530,7 @@ class TimeInput extends React.Component {
 
   handleMeridiemInputKeyDown(event) {
     if (event.keyCode === KeyCode.KEY_LEFT
+      || event.keyCode === KeyCode.KEY_RIGHT
```

Comment:
Moving _backwards_ to the minute input when right arrow is pressed seems like a bad idea. I think we should just say we can't/won't fix this.

Reply:
Another work around would be moving focus out from meridiem to Time Input on right arrow key press : 
```
if (event.keyCode === KeyCode.KEY_RIGHT) {
  this.setState({ meridiemFocused: false });
}
```

this will prevent meridiem from changing on right key press, since focus will be on Time Input and not on meridiem.